boot/startup/syscfg.yml

```diff
@@ -34,6 +34,10 @@ syscfg.defs:
             For bootloader it's main stack, for application this stack is used by interrupts
             and exceptions.
         value: 768
+    MAIN_STACK_FILL:
+        description: >
+            Fill main stack (interrupt stack) with pattern on startup.
+        value: 0
```

Comment:
non-zero value could be used as a pattern

Reply:
Update as requested